src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -65,8 +72,13 @@ private bool ShouldRedrawBufferImage
 				var sizeChanged = false;
 				if (bitmapBuffer != null)
 				{
+#if NET471
+					var s = bitmapBuffer.Size;
+					sizeChanged = s.Width != this.StrokeWidth || s.Height != this.StrokeWidth;
```

Comment:
This looks wrong... The new view width is unequal to the stroke width?

Reply:
it's when we resize form